### PR TITLE
Check for visible at all levels

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

File: src/CairoMakie.jl [hunk 1]
```diff
@@ -565,6 +565,7 @@ function draw_plot(screen::CairoScreen, scene::Scene)
     Cairo.clip(screen.context)
 
     for elem in scene.plots
+        if to_value(get(elem, :visible, true))
```
Comment (yakir12) [inline src/CairoMakie.jl]: add an end and it'll work (tested locally)

File: src/CairoMakie.jl [hunk 2]
```diff
@@ -565,6 +565,7 @@ function draw_plot(screen::CairoScreen, scene::Scene)
...
         draw_plot(scene, screen, elem)
```
Comment (asinghvi17) [inline src/CairoMakie.jl]:
```suggestion
             draw_plot(scene, screen, elem)
        end

**Comment by asinghvi17** [comment]
We actually got rate limited?  😮

**Comment by SimonDanisch** [comment]
That's the github thing in MakieGallery - we probably didn't tag after you took it out?

**Comment by asinghvi17** [comment]
True, I don't think we did.  However, this may also have played a role: https://travis-ci.org/github/JuliaPlots/CairoMakie.jl/builds/666839480#L205-L413

**Comment by jkrumbiegel** [comment]
so can this be merged?

**Comment by asinghvi17** [comment]
Looks like it - and there seem to be no breakages locally

**Pull request merged**